constants.py

```diff
@@ -0,0 +1,27 @@
+# coding: utf-8
+#
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Loads constants for backend use."""
+
+import json
+import os
+
+class Constants(dict):
+    """Transforms dict to object, attributes can be accesed by dot notation"""
+    __getattr__ = dict.__getitem__
+
+with open(os.path.join('assets', 'constants.json'), 'r') as f:
+    CONSTANTS = Constants(json.load(f))
```

Comment:
Ditto for the ``CONSTANTS`` in the frontend. It doesn't need to be all caps, especially since we're just creating angular constants for each of its members.